### ISSUE TITLE
Firefox 69

### ISSUE THREAD

**Issue by dmilith** [overdodactyl/ShadowFox#285]
ShadowFox could set these values under about:config automatically:

1. `toolkit.legacyUserProfileCustomizations.stylesheets` => true
2. `browser.in-content.dark-mode` => true

Without these two ShadowFox will have no effect in Firefox 69

**Comment by ShalokShalom** [comment]
Where can I kiss you? This saved my day. Could someone implement this?

**Comment by rpgdev** [comment]
>     2. `browser.in-content.dark-mode` => true

I'm on firefox 69 and don't seem to need this set to true.

**Comment by Hal47** [comment]
> 
> 
> > ```
> > 2. `browser.in-content.dark-mode` => true
> > ```
> 
> I'm on firefox 69 and don't seem to need this set to true.

You might have already had it on.

**Comment by gen3vra** [comment]
Came here and found this after it wouldn't work for me. This fixed it, thank you.

**Comment by RenatoLopes771** [comment]
One of the devs should pin this on the homepage as a temporary workaround

**Comment by sanyer** [comment]
There is a note about this in wiki, but not on repo's README.md.

**Comment by NotAkitake** [comment]
Many thanks ♥

**Comment by ShalokShalom** [comment]
Why is this not possible to incorporate in the code?

**Comment by NotAkitake** [comment]
It should be, but seems like the project hasn't been updated in forever, can consider it dead.

**Comment by ShalokShalom** [comment]
I see commits?

**Comment by overdodactyl** [comment]
This change would have to be implemented in the updater repo.  See this issue here:

https://github.com/SrKomodo/shadowfox-updater/issues/53

**Comment by toxicbubble** [comment]
Can the info that exists on https://github.com/overdodactyl/ShadowFox/wiki/Installation and an additional line for **browser.in-content.dark-mode => true** be set be added to the "Installation and Updating" section of README.md at least? I spent ages trying to get it to work, and nothing in the installation section of the README.md mentions this. right now it's very unclear why after installation nothing changes.

**Comment by Thynix** [comment]
@toxicbubble Good idea - I just made a fork and [did that](https://github.com/Thynix/ShadowFox#installation-and-updating). I'll look into getting the updater to automatically do it later this week.
